CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
+
+## [Upcoming]
+
+### Models
+
+- Added Aeph Alpha's Luminous models (#1215)
+- Added AI21's J1-Grande v2 beta model (#1177)
+- Added OpenAI's ChatGPT model (#1231)
+- Added OpenAI's text-davinci-003 model (#1200)
+
```

Comment:
Can we also add 

### Adaptation

- Improved sampling of  in-context examples (https://github.com/stanford-crfm/helm/pull/1172)

Reply:
Added.